src/librustc_mir/diagnostics.rs

```diff
@@ -1220,35 +1227,33 @@ let mut i = 0;
 let a = &i; // ok!
 let b = &i; // still ok!
 let c = &i; // still ok!
+b;
+a;
 ```
 "##,
 
 E0500: r##"
-A borrowed variable was used in another closure. Example of erroneous code:
+A borrowed variable was used by a closure. Example of erroneous code:
```

Comment:
are we supposed to be revising the text associated with error messages in this manner? Was this an expected outcome of this change?

Reply:
This is a drive by fix since the documentation for this error is very outdated. The current example gives E0524.